tutorials.markdown

```diff
@@ -133,7 +133,7 @@ id: tutorials
 * [Android reactive programming with RxJava](https://www.youtube.com/watch?v=JCLZ55M2gVo) video of a presentation by Ivan Morgillo
 * [Learning RxJava (for Android) by example](https://newcircle.com/s/post/1744/2015/06/29/learning-rxjava-for-android-by-example) video and speaker deck of a presentation by Kaushik Gopal, and [the associated GitHub repository](https://github.com/kaushikgopal/RxJava-Android-Samples)
 * [Retrofit 2.0: The biggest update yet on the best HTTP Client Library for Android](http://inthecheesefactory.com/blog/retrofit-2.0/en), which now supports RxJava Observables
-* [Fast Android Networking](https://github.com/amitshekhariitbhu/Fast-Android-Networking/blob/master/RxAndroidNetworking.md) Fast Android Networking which is made on top of OkHttp Layer now supports RxJava
+* [Fast Android Networking](https://github.com/amitshekhariitbhu/Fast-Android-Networking/wiki/Using-Fast-Android-Networking-Library-With-RxJava) Big News, Now Fast Android Networking which is made on top of OkHttp Client supports RxJava
```

Comment:
I'd prefer to avoid marketing-style language like "Big News!" "Now!" and that sort of thing.